find-stable-candidates

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -e
+
+if ! type -P xml_grep >/dev/null; then
+	echo "dev-perl/XML-Twig needed for xml_grep"
+	return
+fi
```

Comment:
Let's kill the dep. Plain `git grep` should be good enough, it's not like you put random @g.org emails in metadata.xml, do you? It's already used in `foreach-pkg-maint` which you could perhaps reuse.

Reply:
> it's not like you put random @g.org emails in metadata.xml, do you?

Just a couple, which we can of course fix:

````sh
% find -maxdepth 3 -mindepth 3 -type f -name metadata.xml -exec grep '@gentoo.org' {} + | grep -v email  
./sys-apps/portage/metadata.xml:    <bugs-to>mailto:dev-portage@gentoo.org</bugs-to>
./app-portage/emerge-delta-webrsync/metadata.xml:		<bugs-to>mailto:dev-portage@gentoo.org</bugs-to>
./app-portage/repoman/metadata.xml:    <bugs-to>mailto:dev-portage@gentoo.org</bugs-to>
````

I don't have a strong opinion on whether to keep or remove the `dev-perl/XML-Twig` dep, so I'll remove it if you want but I would note that:

````sh
% emerge -cvp dev-perl/XML-Twig dev-perl/Net-DBus x11-misc/xdg-utils                   

Calculating dependencies... done!
  dev-perl/Net-DBus-1.2.0 pulled in by:
    x11-misc/xdg-utils-1.1.3_p20200220-r5 requires dev-perl/Net-DBus

  dev-perl/XML-Twig-3.520.0-r1 pulled in by:
    dev-perl/Net-DBus-1.2.0 requires dev-perl/XML-Twig

  x11-misc/xdg-utils-1.1.3_p20200220-r5 pulled in by:
    app-office/libreoffice-7.2.5.2-r1 requires x11-misc/xdg-utils
    app-portage/mgorny-dev-scripts-14 requires x11-misc/xdg-utils
    gnome-base/gdm-42.0 requires >=x11-misc/xdg-utils-1.0.2-r3
    gnome-base/gnome-shell-42.0 requires x11-misc/xdg-utils
    media-gfx/simple-scan-42.0 requires x11-misc/xdg-utils
    net-print/cups-2.3.3_p2-r3 requires x11-misc/xdg-utils
````

so we're very likely to have it already—it just requires `USE="gnome"` on `x11-misc/xdg-utils`.

Reply:
Oh, your `git grep -l "${maint}" '**/metadata.xml'` is a lot faster. I experimented with using `git grep` but I didn't quote `**/metadata.xml` so I guess shell expansion is quite slow or something.

Reply:
Yes, it's quite likely it's very slow because it iterates over worktree while (I think) `git grep` matches over index.